sentry/src/main/java/io/sentry/SentryItemType.java

```diff
@@ -62,7 +63,7 @@ public void serialize(final @NotNull ObjectWriter writer, final @NotNull ILogger
     writer.value(itemType);
   }
 
-  static final class Deserializer implements JsonDeserializer<SentryItemType> {
+  public static final class Deserializer implements JsonDeserializer<SentryItemType> {
```

Comment:
do we need this change?

Reply:
Needed it in tests. Also all others are public.